### PR TITLE
Brig Physicians

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Ports brig physicians from Oracle (OracleStation/OracleStation#14)

Also fixes the secHUD icons for paramedics not showing up.

Fully tested and working, all map changes are minor.

## Why It's Good For The Game
It fills a niche not often explored, having someone on hand for both security-related surgeries and general healing of security and prisoners is good. Honestly, I'd play the role myself, and some people have shown interest. I don't think it's job bloat as surgeons were, as this is actually specific and no job was expected to do this before.

## Changelog
:cl:
add: Brig Physicians, for when half of your security department is dying and nobody knows medical.
fix: Paramedics' secHUD icons show up again.
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
fuck this is #1000 

time to jump off a bridge before qwerty gets mad at me for taking it

**Comment by MarkSuckerberg** [comment]
there's some icons modified that shouldn't be, fixing now

**Comment by qwertyquerty** [comment]
Happy #1000

File: code/modules/crew_objectives/security_objectives.dm [hunk 1]
```diff
@@ -28,3 +28,14 @@
 		if(!(M.mind.assigned_role in GLOB.security_positions) && istype(get_area(M), /area/security/prison)) //there's no list of incarcerated players, so we just assume any non-security people in prison are prisoners, and assume that any security people aren't prisoners
 			return FALSE
 	return TRUE
+
+/datum/objective/crew/justicemed
+	explanation_text = "Ensure there are no dead bodies in the security wing when the shit ends."
```
Comment (Moccha-Bee) [inline code/modules/crew_objectives/security_objectives.dm]: "when the shit ends."

Basado

Comment (MarkSuckerberg) [reply code/modules/crew_objectives/security_objectives.dm]: Sometimes the funnies write themselves accidentally.

File: code/modules/jobs/job_types/brigphys.dm [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/datum/job/brig_phys
+	title = "Brig Physician"
+	flag = BRIG_PHYS
+	department_head = list("Head of Security", "Cheif Medical Officer")
```
Comment (ike709) [inline code/modules/jobs/job_types/brigphys.dm]:
```suggestion
	department_head = list("Head of Security", "Chief Medical Officer")
```

Comment (MarkSuckerberg) [reply code/modules/jobs/job_types/brigphys.dm]: Bloody hell, I suck at spelling...

**Comment by hackerjsimon** [comment]
yay 1000

**Comment by 123chess456** [comment]
I knew this is why you delayed this PR for so long

check 'em

also unironically good PR since their roundstart mindshield means that Sec can now actually trust someone to surgerize them.

**Comment by eLeCtrOssSnake** [comment]
1000 Yay! Although I do not really see their gameplay improvement or I just don't understand something. Eh, I am not going to approve nor deny that for now. (I fooled ya)

**Comment by eLeCtrOssSnake** [comment]
Idk, seems legit.

**Review by eLeCtrOssSnake — APPROVED** [review]
Seems legit

**Comment by ike709** [comment]
Remember to update the wiki.

**Comment by hackerjsimon** [comment]
someone will need to edit maps

also the requirements should probs be paramedic and sec guard

**Pull request merged**